using-timescaledb/custom-jobs.md

```diff
@@ -0,0 +1,221 @@
+# Custom Jobs
+
+Custom jobs allow you to run functions and procedures implemented in a
+language of your choosing on a schedule within TimescaleDB. This allows
+automatic periodic tasks that are not covered by existing policies and
+even enhancing existing policies with additional functionality.
+
+### Creating a custom job procedure [](create-procedure)
+
+The signature for custom jobs is (job_id INT, config JSONB).
+The content of the config JSONB is completely up to the job and may
+also be NULL if no variables are required.
+
+Template for a custom job procedure.
+
+```sql
+CREATE OR REPLACE PROCEDURE custom_job(job_id int, config jsonb) LANGUAGE PLPGSQL AS
+$$
+BEGIN
+	RAISE NOTICE 'Executing job % with config %', job_id, config;
+END
+$$;
+```
+
```

Comment:
Perhaps we should have an external pointer to PG documentation here?

https://www.postgresql.org/docs/current/sql-createprocedure.html